docs/tutorials/index.md

```diff
@@ -97,16 +98,19 @@ $ ./bin/start-druid
 [Tue Nov 29 16:31:06 2022] Running command[middleManager]: bin/run-druid middleManager /apache-druid-{{DRUIDVERSION}}/conf/druid/single-server/quickstart '-Xms64m -Xmx64m' '-Ddruid.worker.capacity=2 -Ddruid.indexer.runner.javaOptsArray=["-server","-Duser.timezone=UTC","-Dfile.encoding=UTF-8","-XX:+ExitOnOutOfMemoryError","-Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager","-Xms256m","-Xmx256m","-XX:MaxDirectMemorySize=256m"]'
 ```
 
-All persistent state, such as the cluster metadata store and segments for the services, are kept in the `var` directory under 
-the Druid root directory, apache-druid-{{DRUIDVERSION}}. Each service writes to a log file under `var/sv`.
+It may use up to 80% of the total available system memory.
+To explicitly set the total memory available to Druid, pass a value for the memory parameter. For example, `./bin/start-druid -m 16g`. 
+
+Druid stores all persistent state data, such as the cluster metadata store and data segments, in `apache-druid-{{DRUIDVERSION}}/var`.
+Each service writes to a log file under `var/sv`.
```

Comment:
Did we change that again? In my latest build, logs go to `$DRUID_HOME/log`.

(See how smooth it is to use $DRUID_HOME instead of `apache-druid-{{DRUIDVERSION}}`?

Also, note that here we are prefixing `var` with `apache-druid-{{DRUIDVERSION}}`, where as above we just referenced `bin` without a prefix.